packages/ketcher-core/src/application/formatters/formatterFactory.ts

```diff
@@ -56,6 +60,11 @@ export class FormatterFactory {
       molfileParseOptions.badHeaderRecover = badHeaderRecover
     }
 
+    if (typeof ignoreChiralFlag === 'boolean') {
+      molfileParseOptions.ignoreChiralFlag = ignoreChiralFlag
+      structServiceOptions['ignore-no-chiral-flag'] = ignoreChiralFlag
```

Comment:
Just curios about the flag name "ignore-no-chiral-flag" - is it already supported by Indigo?